ipt_connect/IPTdev/templates/IPTdev/tournament_overview.html

```diff
@@ -16,13 +16,20 @@ <h1>{% trans "Welcome!" %}</h1>
         </td>
     {% endif %}
     <td>
-        <p class="emphase">{% trans "This interface gathers the results of the " %}<a href="http://dev.iptnet.info">{{params.NAME.front}}</a>.</p>
+        <p class="emphase">
+            {% trans "This interface gathers the results of the " %}
+            <a href="{{ params.website_url }}">{{params.NAME.front}}</a>.
+        </p>
 
         <p class="emphase">{% trans "Most of the displayed elements can be clicked on. You can access teams, individual participant or jury member statistics, as well as details of each Round, Physics Fights and Problem." %}</p>
 		
-		<p class="emphase">{% trans "The results are updated continuously as the Physics Fights go on, resulting in a display that might not be completely accurate 100% of the time (due to grade checking). Please check your emails and " %}<a href="{{ link }}">{% trans "the website" %}</a>{% trans " for the official announcements of the results after each Fight." %}
-
         <p class="emphase">{% trans "In case you spot a bug or have any suggestion to improve the software, please get in touch with us at webmaster (a) iptnet (dot) info." %}</p>
+		<p class="emphase">
+            {% trans "The results are updated continuously as the Physics Fights go on, resulting in a display that might not be completely accurate 100% of the time (due to grade checking). Please check your emails and " %}
+            <a href="{{ params.website_url }}">{% trans "the website" %}</a>
+            {% trans " for the official announcements of the results after each Fight." %}
```

Comment:
Since we are now using the update_all function to update the score, this message is not really correct anymore. I'd replace with something like 
"The results are updated once after each round - you will be notified via email once the update has been made"

Reply:
Moving the discussion to #97